docs/reference/commandline/daemon.md

```diff
@@ -898,6 +900,8 @@ This is a full example of the allowed configuration options in the file:
 	"cluster-store": "",
 	"cluster-store-opts": [],
 	"cluster-advertise": "",
+	"max-concurrency-download": 3,
+	"max-concurrency-upload": 5,
```

Comment:
Yes, this doesn't match any of the other variable names.

Runtime config would be cool, but that obviously has to come from the client side.